h2o-docs/src/product/data-science/model_selection.rst

```diff
@@ -217,8 +258,15 @@ Examples
       Intercept       CAPSULE           AGE          RACE         DPROS         DCAPS           PSA           VOL
       4.8526636043  0.7153633278  0.0069487980 -0.0584344031  0.0791810013  0.4353149856  0.0126060611  -0.0005196059
 
+      # Retrieve the list of coefficients for a subset size of 3:
+      coeff3 <- h2o.coeff(allsubsetsModel, 3)
+      print(coeff3)
+      [[3]]
+      Intercept    CAPSULE      DCAPS        PSA
+      5.34902149 0.75750144 0.47979555 0.01289096
+
       # Retrieve the list of standardized coefficients:
-      coeff_norm <- h2o.coef_norm(maxrglmModel)
+      coeff_norm <- h2o.coef_norm(allsubsetsModel)
       [[1]]
```

Comment:
please add accessing coef_norm for a single subset size as well like h2o.coef_norm(allsubsetsModel, 3)